.github/workflows/ci.yaml

```diff
@@ -70,7 +68,28 @@ jobs:
       #     sudo apt-get install curl
       #     curl -L https://github.com/hasura/graphql-engine/raw/stable/cli/get.sh | bash
 
-      - name: Smoke tests
+      - name: Run migrations on local container
         # TODO: check if hasura migrations + hasura ic list
         run: |
           DATABASE_URL=postgres://postgres:password@localhost:5432/minterop diesel migration run
+
+      - name: Apply hasura metadata to sandbox
+        run: |
+          npm i -g hasura-cli --y
+          hasura metadata apply --project hasura --admin-secret "${{ secrets.HASURA_SANDBOX_ADMIN }}" --endpoint https://interop-sandbox.hasura.app/
```

Comment:
Why not use SANDBOX_ENV?

Reply:
Saw that in secrets as well, this was just copied from branch on mintlake, if you feel strongly we should keep the whole env consistent will change it